### DISCUSSION TITLE
What is the occupation of Mary Poppins?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#263] [Galaxy Trivia] [answered]
**Hint:** 👶🍼👩

**Comment by live627** [comment]
I'm guessing that the combination of icons means babysitter/nana.

**Comment by thuongtruong1009** [comment]
Mary Poppins is a fictional character who is often portrayed as a magical nanny. In the book series by P.L. Travers and in the Disney film adaptations, Mary Poppins is hired as a nanny by the Banks family to care for their children. Therefore, her occupation is that of a nanny or governess. However, her unique abilities and magical powers often allow her to go above and beyond the typical duties of a nanny.

**Comment by RexRowan** [comment, marked as answer]
nanny
